packages/flutter/test/material/tabs_test.dart

```diff
@@ -386,6 +386,40 @@ void main() {
       material3 ? const Size(14.25, 72.0) : const Size(14.0, 72.0));
   });
 
+  testWidgets('Material2 - Tab icon-label margin', (WidgetTester tester) async {
+    final ThemeData theme = ThemeData(useMaterial3: false);
+    Widget buildApp() {
+      const Tab tab = Tab(icon: Icon(Icons.house), text: 'x');
```

Comment:
```suggestion

```
Does this have to be a variable? if it isn't reused, we can directly put this `Tab` in the `child` parameter and format the code accordingly

Reply:
Not necessarily, but I feel it reads better than the following, which is the test right above. The variable separates what's being tested.

```dart
await tester.pumpWidget(
    MaterialApp(theme: theme, home: const Center(child: Material(child: Tab(icon: SizedBox(width: 10.0, height: 10.0), child: Text('x'))))),
);
```

Would this be preferred?
```dart
return MaterialApp(
  theme: theme,
  home: const Center(child: Material(
    child: Tab(icon: Icon(Icons.house), text: 'x'),
  ),
));
```

Reply:
The second is better, add a line break for `text`.

Please read the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo) for this PR and future PRs.

Reply:
Done in a889db9

I did look into the style guide but I didn't find anything related to widget trees. Did I miss something? Apologies if that wasn't the expected format. I've seen in the tests that widget trees tend to be inlined to reduce the number of lines in the test files, unlike a "regular" Flutter app with dartfmt.

Thank you for the reviews on this and other PRs!